src/pushi/base/web_push.py

```
@@ -245,38 +303,37 @@ def send(self, app_id, event, json_d, invalid={}):
-                    except Exception as delete_exception:
-                        self.logger.error(
-                            "Failed to delete expired subscription: %s"
-                            % str(delete_exception)
-                        )
+                    self.logger.info("Subscription expired: '%s'" % endpoint)
+                    sub_obj = sub.get("_obj")
+                    if sub_obj:
+                        try:
+                            sub_obj.delete()
+                        except Exception:
+                            pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)